vagrant-pxe-harvester/ansible.cfg

```diff
@@ -1,3 +1,4 @@
 [defaults]
 stdout_callback = yaml
 interpreter_python = auto_silent
+display_skipped_hosts = False
```

Comment:
What is this config for?
Looks like redundant. I cannot find anything related to this.

Reply:
It's to decrease console verbosity cuz ther are more and more conditional branch.
It let ansible not to show skipped tasks.

Ref. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/default_callback.html#parameter-display_skipped_hosts

Reply:
Thanks for the explanation. I have no question!